doc/man/repoq.1.rst

```diff
@@ -0,0 +1,182 @@
+.. vim: ft=rst sw=2 sts=2 et
+
+==========
+ **repoq**
+==========
+
+------------------------------------------------
+Output repository information for given products
+------------------------------------------------
+
+:Author: Roman Neuhauser <rneuhauser+repose@sigpipe.cz>
+:Date: Feb 04, 2016
+:Copyright: GPL-2.0
+:Version: 0.28
+:Manual section: 1
+:Manual group: BSD General Commands Manual
```

Comment:
This document can have any license its creator chose. I'm just converting it to rst. If you run now man repose on your laptop you will see 'BSD General Commands Manual'. I'm not sure where it gets it from since I cannot see it in the .in file.